src/components/notifications/CeredSMS.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div class="mb-3">
+        <label for="ceredsms-key" class="form-label">{{ $t('API Key') }}</label>
+        <HiddenInput id="ceredsms-key" v-model="$parent.notification.ceredsmsApiKey" :required="true" autocomplete="new-password"></HiddenInput>
```

Comment:
Could you add a link to the docs or where users can find this token as a helptext?